model/device.go

```diff
@@ -0,0 +1,65 @@
+// Copyright 2020 Northern.tech AS
+//
+//    Licensed under the Apache License, Version 2.0 (the "License");
+//    you may not use this file except in compliance with the License.
+//    You may obtain a copy of the License at
+//
+//        http://www.apache.org/licenses/LICENSE-2.0
+//
+//    Unless required by applicable law or agreed to in writing, software
+//    distributed under the License is distributed on an "AS IS" BASIS,
+//    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//    See the License for the specific language governing permissions and
+//    limitations under the License.
+
+package model
+
+import (
+	"time"
+)
+
+type DeviceAttribute struct {
+	Name        string      `json:"name" bson:",omitempty"`
+	Description *string     `json:"description,omitempty" bson:",omitempty"`
+	Value       interface{} `json:"value" bson:",omitempty"`
+	Scope       string      `json:"scope" bson:",omitempty"`
+}
+
+// Inventory device wrapper
+type InvDevice struct {
+	//system-generated device ID
+	ID string `json:"id" bson:"_id,omitempty"`
+
+	//a map of attributes names and their values.
```

Comment:
This comment doesn't apply.

Reply:
yeah, cool, but it still is from the ent counterpart. maybe lets leave it for the sake of next re-back-port-merge to enterprise?